codec/std/codec.proto

```diff
@@ -47,8 +47,9 @@ message Supply {
 // Evidence defines the application-level allowed Evidence to be submitted via a
 // MsgSubmitEvidence message.
 message Evidence {
-  option (gogoproto.equal)             = true;
-  option (cosmos_proto.interface_type) = "github.com/cosmos/cosmos-sdk/x/evidence/exported.Evidence";
+  option (gogoproto.equal) = true;
+  option (cosmos_proto.interface_type) =
```

Comment:
are you formatting on save? @alexanderbez suggested the following settings for VSCode:

```
"clang-format.style": "{BasedOnStyle: Google, IndentWidth: 2, ColumnLimit: 0, AlignConsecutiveAssignments: true, AlignConsecutiveDeclarations: true, SpacesInSquareBrackets: true}",
```

Reply:
I was using the default settings, may be a good idea to specify the format settings otherwise these will get edited by many people